app/assets/javascripts/discourse/app/services/key-value-store.js

```diff
@@ -0,0 +1,32 @@
+import Service from "@ember/service";
+import KeyValueStore from "discourse/lib/key-value-store";
+
+const PROXIED_METHODS = [
+  "abandonLocal",
+  "remove",
+  "set",
+  "setObject",
+  "get",
+  "getInt",
+  "getObject",
+  "getItem",
+  "removeItem",
+  "setItem",
+];
```

Comment:
Should we make the list dynamically, using `Object.getOwnPropertyNames(KeyValueStore.prototype)`, so that we won't have to worry about it getting out of sync? (unlikely, but still)

Reply:
Nice! I had tried to use `Object.entries(KeyValueStore.prototype)` but it didn't work because (most of) the functions are not itterable. `getOwnPropertyNames` works great, although we do need to manually filter out the 'constructor' method.